src/common/iop_order.h

```diff
@@ -0,0 +1,193 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2018 edgardo hoszowski.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef DT_IOP_ORDER_H
+#define DT_IOP_ORDER_H
+
+#ifdef HAVE_OPENCL
+#include "CL/cl.h"           // for cl_mem
+#endif
+
+struct dt_iop_module_t;
+struct dt_develop_t;
+struct dt_dev_pixelpipe_t;
+
+typedef struct dt_iop_order_entry_t
+{
+  double iop_order;
+  char operation[20];
```

Comment:
current max for an operation name is 16, 20 seems ok but a bit short maybe?

Reply:
this is a
https://github.com/darktable-org/darktable/blob/master/src/control/settings.h#L35

seems that this is new, I'll change it.